.github/workflows/pyomslnxwheel.yml

```diff
@@ -382,27 +382,27 @@ jobs:
 
     # TODO: add depending on matrix.os; Download artifacts
     - name: Download acrtifacts from macwheels
-      if: runner.os == 'windows-latest'
+      shell: bash 
+        if: runner.os == 'Windows'
+        run: |  
+          wget --no-parent -r -P $HOME/macwheels_abi "https://abibuilder.informatik.uni-tuebingen.de/archive/openms/pyopenms/macOS/"  
         #uses: actions/download-artifact@v1
         #with:
           #name: winwheels 
-        shell: bash 
+
+        if: runner.os == 'macOS'
         run: |  
           wget --no-parent -r -P $HOME/macwheels_abi "https://abibuilder.informatik.uni-tuebingen.de/archive/openms/pyopenms/macOS/"  
-      if: runner.os == 'macos-latest'
```

Comment:
Why not use the runner.os variable to construct the correct link? Then you only need one step.

Reply:
I have seen that - similar to your miniconda code, but I am unsure it that works as well if we want to use the "download/artifacts" later on.

Reply:
I see. I would hope it works like `artifact: name: ${{runner.os}}-wheels` but you never know with their crap actions.